### DISCUSSION TITLE
How to record log activity on relation manager?

### DISCUSSION THREAD

**Discussion by AFSDARIF** [Z3d0X/filament-logger#67] [Q&A]
I want relation manager on resource to be recorded on log activity. Thanks before.

**Comment by Z3d0X** [comment]
Currently this doesn't work out of the box.
However you could use register relation manager models under model logger https://github.com/Z3d0X/filament-logger/blob/main/config/filament-logger.php#L38

There is a possibility that relation manager logging out-of-the-box to be added in the future

**Reply by dharen008** [reply]
You should add this to the installation instructions. If I hadn't found this post, I would have added the code to the model file as shown in the Spatie documentation.
